### DISCUSSION TITLE
My skill has no effect, here's how to defeat him

### DISCUSSION THREAD

**Discussion by Olegarh1a** [ihhub/fheroes2#8759] [General]
### Your question

A month of play has passed. killed him 2 strong heroes, and then the question is how to get through if he still has a lot of strong ones

### Additional info

![1](https://github.com/ihhub/fheroes2/assets/7952252/9dad6be0-8a7c-47d8-a596-9d546d582dca)
![2](https://github.com/ihhub/fheroes2/assets/7952252/8c1e81af-99cd-42d8-9f9b-3ca1f689b3e8)
![3](https://github.com/ihhub/fheroes2/assets/7952252/add863a2-792b-430b-a3d9-aa70634a0e90)
![4](https://github.com/ihhub/fheroes2/assets/7952252/45d24252-a224-4609-9a18-9e7348d8f786)

**Comment by Alucard648** [comment]
That is his weak heroes, you later encounter nigh impossible necromancer with 4k skeletons, 400 vampire lords, 200 Bone Dragons, Berserk,  Dimension door and huge stats.

**Comment by Olegarh1a** [comment]
> Это его слабые герои, позже вы столкнетесь с почти невозможным некромантом с 4 тысячами скелетов, 400 лордами-вампирами, 200 костяными драконами, Берсерком, дверью измерений и огромной статистикой.

So he is closed and sits without touching, yellow and green are actively running around, in the first days we cleaned the islands with resources, I am without resources. In the old heroes, the computer didn’t sail to the islands, God, it didn’t even collect resources near the people’s castle. In a month when he collects 40 cyclops, well, forgive me. I play on normal difficulty

**Comment by Olegarh1a** [comment]
and he also has a hero running around with another 25+ Black dragons

**Comment by Alucard648** [comment]
> > Это его слабые герои, позже вы столкнетесь с почти невозможным некромантом с 4 тысячами скелетов, 400 лордами-вампирами, 200 костяными драконами, Берсерком, дверью измерений и огромной статистикой.
> 
> So he is closed and sits without touching, yellow and green are actively running around, in the first days we cleaned the islands with resources, I am without resources. In the old heroes, the computer didn’t sail to the islands, God, it didn’t even collect resources near the people’s castle. In a month when he collects 40 cyclops, well, forgive me. I play on normal difficulty

That is strange. In my attempts, he uses Dimension Door to rush me before end of week 4. Building Titans that early is one huge acievment. Opponent offs my entire army with a single Death Wave cast.

**Comment by Olegarh1a** [comment]
Sorry, I can't open the map. showed all the heroes: titans, black dragons, cyclops, phoenixes. 3-4 heroes who cannot be killed in the second month.
AI is too strong on normal

**Comment by ihhub** [comment]
This is not an issue. I am moving it to Discussion section.

**Comment by fheroes2bugs** [comment]
What is the map name? I need to try it.

Heroes2 was not made perfectly balanced, it was made for fun (remember sphinx? skill will not help you). Remember the map "Arrax The Jerk"? There is no garantee you can win the map even on easy difficulty.
For example, the original map, most popular map in heroes 2 - Broken Alliance. You can find it in beta and in every copy of the game. It was made by creators of the game. Is it balanced? You can be very hard skilled, and even set "game difficulty" to "easy", but if purple player from the start got "dimension door" and "town portal" he will be jumping around the whole map at 2nd week with 14 titans and other troops, that can't be beaten – you can just run around the map and pray that purple will not choose you as his aim №1.

And don't forget - each map has it's own "map difficulty"! 
![image](https://github.com/ihhub/fheroes2/assets/145407420/45a58c8c-c08f-4b5a-8520-841d4873f465)


Some maps (especially maps with "map difficulty" - "expert") just can't be completed by regular players, even if you put "game difficulty" on easy!

**Reply by Olegarh1a** [reply]
the price of loyalty - mission 8

**Reply by fheroes2bugs** [reply]
> the price of loyalty - mission 8

it has name "Corrpution's Heart"

try to put easy campaign difficulty:

![image](https://github.com/ihhub/fheroes2/assets/145407420/94c007fc-2e3e-43e1-b00e-7b41fd348c65)


I can see some players have already done it in previous version of fheroes2. May be you could watch the video and learn some tricks, that will help you:

https://www.youtube.com/watch?v=OOkwhJMixA8 and https://www.youtube.com/watch?v=IE_zNZkcgAs

**Reply by Olegarh1a** [reply]
https://www.youtube.com/watch?v=OOkwhJMixA8 

The computer is significantly different, and note that this is a hard mode. 

That's the difference between Hard mode and normal
1. This is my walkthrough(normal)
2. Month:1 week:2 day:5
![1](https://github.com/ihhub/fheroes2/assets/7952252/b769ac17-f9a1-402f-896c-85b61b27bc7b)
![2](https://github.com/ihhub/fheroes2/assets/7952252/628d775e-79e4-41c7-8ff3-f5d90aa6a086)

and on video https://www.youtube.com/watch?v=OOkwhJMixA8  (hard)

this picture is from the video. Time on video is 9:04. The AI ​​clearly has a weak army
Month:1 week:2 day:5

![6](https://github.com/ihhub/fheroes2/assets/7952252/34b72abe-854a-443f-a05e-6c0f59ee3513)

![4](https://github.com/ihhub/fheroes2/assets/7952252/f85e870c-494f-4238-95d1-2086bbd26b3c)


3. there is a difference in starting resources
4. My start 
5. https://www.youtube.com/watch?v=sOnmpVCIvdU&ab_channel=ClassicGuru

**Reply by fheroes2bugs** [reply]
The starting resources is very strange moment. Both you and these 2 gamers videos I gave to you, play hard difficulty, but you play with starting 2500 gold, but in the video they both have 7500 gold.

I can see this 2 gamers invade green castle at week 4 and week 2, but there is no bridge and pyramid built. Looks like in new version of fheroes2 computer on hard difficulty got extra resources... Because at your video at week 2 you fight vs 7 cyclops.

Looks like computer did not get only stronger AI then in the last version, he also gets more resources, that is not fair.

Also, can you please relate the save file "POL_6_Coplete"? I need to try it. Pack it to zip and drag&drop into the message.

**Reply by Olegarh1a** [reply]
[POL_6_Complete.zip](https://github.com/user-attachments/files/15507901/POL_6_Complete.zip)

yellow and purple, they also develop abnormally quickly

**Reply by fheroes2bugs** [reply]
> [POL_6_Complete.zip](https://github.com/user-attachments/files/15507901/POL_6_Complete.zip)
> 
> yellow and purple, they also develop abnormally quickly

Thank you.

I tried it. I can't select "hard mode" - so I play on "normal". Is this ok? I got the same starting resources as you do (2500 gold). My result is - i captured that green barbarian castle at week 1, day 7. In fact, I think after a few more tries, it could be done on day 6. At week 2 day 1 I will hire trolls.
The knight castle seems easy to be taken at week 1 also. Both of them must be taken at the end of the week 1.

The trick is - you need to capture as fast as you can - you don't need to pick all the resources etc. The more you sit on your base - the stronger enemy gets and you would not be able to win him...

Here is the video:

https://github.com/ihhub/fheroes2/assets/145407420/43e7c793-1a07-4058-a0c4-1c4bf39b9dba



Do you want full replay from me?

**Reply by Olegarh1a** [reply]
Yes it would be nice

**Reply by fheroes2bugs** [reply]
> Yes it would be nice

@Olegarh1a  here is someone doing this map in fheroes: https://www.youtube.com/watch?v=83mwVae2y6g
don't know what version he is playing, but video is only 1 week old

**Reply by Olegarh1a** [reply]
the difference is very big, for 3 weeks the computer does not have strong creatures, I haven’t played for a long time, maybe because of updates

**Reply by Olegarh1a** [reply]
strange behavior of green: he is too passive. purple doesn't try to get out of his lair either

**Reply by Olegarh1a** [reply]
first month week 4 day 5
There are 4 cyclops in the green army.
in my video week 2 day 5 there are already 7 cyclops 14 trolls on the computer. 
I repeat, the difference is too big, maybe it’s worth updating and trying

**Reply by fheroes2bugs** [reply]
> maybe because of updates

he is playing version 1.1.1 or earlier

I tried to to play this version (1.1.1) with save-file, that you gave me. I was trying to do my best for the first week. By the end of the week 1 - I captured 2 castles:
![2024-09-17_162342](https://github.com/user-attachments/assets/061b7989-4e89-42ce-99e0-800a0c2a7ac7)
Will try to play week 2 and week 3 and will see.

If I try to land on the north territory later than day 4, computer will have cyclops and trolls. The time is very, very importand in this mission.

Also, at the video, you can see that elfs are joining at the starting island - that is very important too.
